### ISSUE TITLE
broken pre-flight validation in `con.save_user_defined_process`

### ISSUE THREAD

**Issue by soxofaan** [Open-EO/openeo-python-client#526]
The pre-flight validation in `con.save_user_defined_process` isn't working.

```python
import openeo
import openeo.processes
con = openeo.connect("openeo.vito.be")
con.authenticate_oidc()
con.save_user_defined_process(
    user_defined_process_id="add35",
    process_graph=openeo.processes.add(3, 5),
)
```
gives `logging.error()`:
```
Preflight process graph validation failed: 'process'
```

Luckily, the actual UDP storing is not blocked

**Comment by soxofaan** [comment]
(reported here https://discuss.eodc.eu/t/udp-creation/669 )

**Comment by soxofaan** [comment]
fixed by 2dc9c44979be309712b7690b1b1d65bc0b9ec7da
